corehq/apps/hqadmin/templates/hqadmin/admin_restore.html

```diff
@@ -40,13 +45,33 @@
                     <div>
                         Number of locations in this restore: <strong>{{num_locations}}</strong>
                     </div>
+                    {% if location_type_counts %}
+                        <div>
+                            Number of locations by location type
+                        </div>
+                    {% endif %}
                     <div>
                         {% for location_type, count in location_type_counts.items %}
                             <div>
                                 &emsp;{{location_type}}: {{count}}
                             </div>
                         {% endfor %}
                     </div>
+                    <div>
+                        Number of reports in this restore: <strong>{{num_reports}}</strong>
+                    </div>
+                    {% if report_row_counts %}
+                        <div>
+                            Number of rows by report
+                        </div>
+                    {% endif %}
+                    <div>
+                        {% for report_id, row_count in report_row_counts.items %}
+                            <div>
```

Comment:
is there a reason you didn't use a `<ul><li>` for these?

Reply:
Nope, changed